lib/ex_aws/ec2.ex

```diff
@@ -21,7 +21,7 @@ defmodule ExAws.EC2 do
   ExAws.EC2.describe_instances(filters: [image_id: "ami-1ecae776"])
   ExAws.EC2.describe_instances(filters: ["network-interface.availability-zone": "us-east-1a"])
   ExAws.EC2.describe_instances(filters: ["tag:elasticbeanstalk:environment-name": "demo"])
-  ExAws.EC2.describe_instance_status(instances: ["i-e5974f4c"])
+  ExAws.EC2.describe_instance_status(instance_id: ["i-e5974f4c"])
```

Comment:
Certainly doable, this first stab was naming them to match the API parameters.

Without adding multiple enumerations through the options these could all be handled in the `normalize_opts` reduce with either of these...

  ```@pluralized_parameters [
    "instance_id",
    "block_device_mapping",
    ...
  ]

  @pluralized_parameters
  |> Enum.map(fn param -> defp reduce_list_params({unquote(String.to_existing_atom("#{param}s")), val}, acc) when is_list(val), do: Map.merge(acc, list_builder(val, unquote(String.to_atom(param)), 1, %{})) end)```

or defining each in full
```
defp reduce_list_params({:instance_ids, keys}, acc), do: Map.merge(acc, list_builder(val, :instance_id, 1, %{}))
  defp reduce_list_params({:block_device_mappings, keys}, acc), do: Map.merge(acc, list_builder(val, :block_device_mapping, 1, %{}))
  ```

Do either of those sound reasonable or should they all be handled in the function? This is how I was handling that before:

```
instance_ids = Keyword.get(opts, :instances, [])
    query_params = opts
    |> Keyword.delete(:instances)
    |> normalize_opts
    |> Map.merge(%{
      "Action"  => "DescribeInstanceStatus",
      "Version" => @version
      })
    |> Map.merge(list_builder(instance_ids, "InstanceId", 1, %{}))
```